docs/projects.md

```diff
@@ -0,0 +1,85 @@
+# Projects
+
+The following is a list of projects that this project will support and develop. The following projects fit together to form the 
+Executable Book Project. 
+
+```{contents} Contents:
+```
+
+## Overview
+
+{{ add Version 3 build diagram }}
+
+High level feature lists are provided as a simple measure of progress. `Phase 1` are items required for minimal viable product, `Phase 2` include items that are planned for initial release, and 
+`Phase 3` captures wishlist items to be implemented in the future.
+
+## Authoring Tools
+
+### [JupyterText](https://github.com/mwouts/jupytext) [External]
+
+**Description:** 
+
+Jupyter Notebooks as Markdown Documents, Julia, Python or R scripts
+
+**Contribution:** 
+
+The primary aim is to add myst to jupytext to enable *bi-directional* conversion 
+between IPYNB and myst text based files. 
+
+A seconday objective is to understand how JupyText may be used to convert between 
```

Comment:
```suggestion
A secondary objective is to understand how JupyText may be used to convert between 
```

Reply:
@chrisjsewell what do you use for spell check in `vscode` -- shamelessly going to copy your dev environment :-)

Reply:
Haha, well I use [code-spell-checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker). But if you want to copy me completely then all my setting are synced to GitHub (with [settings-sync](https://marketplace.visualstudio.com/items?itemName=Shan.code-settings-sync)):
https://gist.github.com/chrisjsewell/0e808e0f42c1a0b02d393df767027722 😄

Reply:
well that's awesome :-) I just came across this https://marketplace.visualstudio.com/itemdetails?itemName=Shan.code-settings-sync this week. Looks awesome.